### PR TITLE
Add FileInfo and get file []byte in plugin api

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Add support in the plugin API to get the FileInfo and the []byte for the file

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Added or updated unit tests (required for all new features)
- [ ] Added API documentation (required for all new APIs)

### CODE REVIEW

File: app/plugin_api.go [hunk 1]
```diff
@@ -279,6 +279,14 @@ func (api *PluginAPI) CopyFileInfos(userId string, fileIds []string) ([]string,
 	return api.app.CopyFileInfos(userId, fileIds)
 }
 
+func (api *PluginAPI) GetFileInfo(fileId string) (*model.FileInfo, *model.AppError) {
+	return api.app.GetFileInfo(fileId)
+}
+
+func (api *PluginAPI) ReadFileAtPath(path string) ([]byte, *model.AppError) {
```
Comment (jwilander) [inline app/plugin_api.go]: Wondering if it would be better to just name it `ReadFile`?

Comment (enahum) [reply app/plugin_api.go]: Want me to do the change? I'm 0/5 really

Comment (jwilander) [reply app/plugin_api.go]: Yeah let's change it

**Comment by enahum** [comment]
@jwilander changes made

**Comment by jasonblais** [comment]
Auto-generated docs

**Pull request merged**